presto-product-tests/src/main/resources/sql-tests/testcases/intersect/intersectAndUnion.result

```diff
@@ -0,0 +1,7 @@
+-- delimiter: |; ignoreOrder: true;
```

Comment:
If you want to you can group all these 4 convention files into one. It can be done in the same way as: https://github.com/prestodb/tempto/blob/master/tempto-examples/src/main/resources/sql-tests/testcases/nation/allRows.sql